core/layout_api.php

```diff
@@ -91,18 +93,26 @@ function layout_page_header_begin( $p_page_title = '' ) {
 		echo "\t", '<link rel="shortcut icon" href="', helper_mantis_url( $t_favicon_image ), '" type="image/x-icon" />', "\n";
 	}
 
-	# Advertise the availability of the browser search plug-ins.
-	$t_title = htmlspecialchars( config_get( 'search_title' ) );
-	$t_searches = array( 'text', 'id' );
-	foreach( $t_searches as $t_type ) {
+	if( auth_anonymous_enabled() ) {
```

Comment:
Should you use `current_user_is_anonymous()` instead? Is the sitemap useful for signed in users?

Reply:
>  Is the sitemap useful for signed in users?

Yes, sitemap is for search engines only. They'll never log in anywhere.